base/deprecated.jl

```diff
@@ -112,6 +112,18 @@ macro deprecate_binding(old, new, export_old=true)
          Expr(:call, :deprecate, __module__, Expr(:quote, old)))
 end
 
+macro deprecate_moved(old, new, export_old=true)
+    eold = esc(old)
+    return Expr(:toplevel,
+         :(function $eold(args...; kwargs...)
+               error($eold, args, " has been moved to the package ", $new, ".jl.\n",
```

Comment:
I actually think we should omit the `args` here; what you've passed could be an invalid method for the function, so it's misleading to say that the exact thing you passed has moved to a different package. Instead I think we should just say that the function has moved, i.e. `error($eold, " has been moved...")`.